src/System Application/App/Translation/src/TranslationBuffer.Table.al

```diff
@@ -0,0 +1,59 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Globalization;
+
+table 3713 "Translation Buffer"
```

Comment:
why aren't you re-using the translation table as a temporary?

Reply:
a simple IsTemporary when calling the method would solve that issue no? Anyways, just double checked and the Translation table is internal so you wouldn't be able to use it unless we made it public... guess it's fine as is then...